PyDynamic/uncertainty/propagate_filter.py

```diff
@@ -80,10 +82,18 @@ def FIRuncFilter(y, sigma_noise, theta, Utheta=None, shift=0, blow=None, kind="c
         elif kind == "corr":
             sigma2 = sigma_noise
         else:
-            raise ValueError("unknown kind of sigma_noise")
+            raise ValueError(
+                "Unknown kind `{KIND}`. Don't now how to interpret the array sigma_noise.".format(
```

Comment:
"know"

Reply:
Maybe we want to add the hint on the allowed cases like :"Please choose `diag` or `corr`." or more generic and less error prone after refactorings and BREAKING CHANGES: "Please refer to the documentation for the allowed values."